.nuspec/maui-blazor.aotprofile.txt

```diff
@@ -1268,7 +1243,6 @@ Methods:
 	intptr Android.Runtime.JNIEnv:GetStaticFieldID (intptr,string,string)
 	intptr Android.Runtime.JNIEnv:GetStaticObjectField (intptr,intptr)
 	intptr Android.Runtime.JNIEnv:GetSuperclass (intptr)
-	intptr Android.Runtime.JNIEnv:monodroid_timing_start (string)
```

Comment:
This means I accidentally recorded the previous profile with `debug.mono.log` set to some value.

I pushed a change to clear this system property for next time.